### PR TITLE
Test signup

### PR DESCRIPTION
**Pull request by godwinpang**

### CODE REVIEW

File: src/tests/AuthController.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+describe('signUpUser', () => {
+  it('should return undefined on nonpermitted email', async () => {
+    (AppUserServiceImpl.getAppUserByEmail as jest.Mock).mockImplementation(() => undefined);
+
+    expect(await authController.signUpUser(new AppUserSignupRequest())).toEqual(undefined);
+  });
+
+  it('should return undefined for guest', async () => {
+    (AppUserServiceImpl.getAppUserByEmail as jest.Mock).mockImplementation(() => new AppUser());
+    (AppUserServiceImpl.isGuest as jest.Mock).mockImplementation(() => undefined);
```
Comment (thaigillespie) [inline src/tests/AuthController.test.ts:29]: Should return true

**Review by thaigillespie — APPROVED** [review]
Lgtm

File: src/tests/AuthController.test.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+describe('signUpUser', () => {
+  it('should return undefined on nonpermitted email', async () => {
+    (AppUserServiceImpl.getAppUserByEmail as jest.Mock).mockImplementation(() => undefined);
+
+    expect(await authController.signUpUser(new AppUserSignupRequest())).toEqual(undefined);
+  });
+
+  it('should return undefined for guest', async () => {
+    (AppUserServiceImpl.getAppUserByEmail as jest.Mock).mockImplementation(() => new AppUser());
+    (AppUserServiceImpl.isGuest as jest.Mock).mockImplementation(() => false);
```
Comment (thaigillespie) [inline src/tests/AuthController.test.ts:29]: Shouldn't this be true and not false?

**Review by thaigillespie — CHANGES_REQUESTED** [review]

**Comment by godwinpang** [comment]
Yeah i suck

**Comment by thaigillespie** [comment]
lmao you're fine

**Review by thaigillespie — APPROVED** [review]

**Pull request merged**